src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -853,8 +852,7 @@ namespace winrt::Microsoft::Terminal::Control::implementation
     //   concerned with initialization process. Value forwarded to event handler.
     void ControlCore::_updateFont(const bool initialUpdate)
     {
-        const auto newDpi = static_cast<int>(static_cast<double>(USER_DEFAULT_SCREEN_DPI) *
-                                             _compositionScale);
+        const auto newDpi = static_cast<int>(lrint(_compositionScale * USER_DEFAULT_SCREEN_DPI));

```

Comment:
this cast seems extraneous or at the very best silly given the `auto`

Reply:
Otherwise AuditMode complains that `long` isn't automatically convertible to `int` (the parameter type for `SetDPI`).